### ISSUE TITLE
@include and temporary tables

### ISSUE THREAD

**Issue by boesi** [dbeaver/dbeaver#7826]
#### System information: 
- Operating system (distribution) and version: Windows 10 1909
- DBeaver version: 6.3.4.202002011957

#### Connection specification:
- Database name and version MySQL 5.6

#### Describe your question:
I have created some basic sql scripts that I use to filter objects from my database. In these scripts I create temporary tables, and when I include them in my actual script using @include, I want to access the temporary table. So I only have to write the filter once and can use it in my different scripts.

This worked without problems in older versions of dbeaver, but not anymore. It seems as if the included script is executed in a separate connection to the database, so the actual script can't access the temporary table.

What am I doing wrong here? Is there some setting to force the desired behaviour?

**Comment by kseniiaguzeeva** [comment]
Sorry for the long answer. Is it still actual for you?
Is it possible to give me Script example?

**Comment by boesi** [comment]
Yes it is still actual for me. I'll build a little example next week.

**Comment by kseniiaguzeeva** [comment]
Thank you. Will wait for

**Comment by uslss** [comment]
Issue hasn't been updated more than a month. If it is still actual for you - feel free to reopen the ticket with additional info that could help us to investigate the issue.

**Comment by boesi** [comment]
Sorry it took me so long to answer.

In the meantime I use DBeaver version 7.1.0.202005311732 and the behaviour changed a bit.

I have two files:

**use.sql**

    @include "create.sql"
    SELECT v1, v2 FROM temp_table;

**create.sql**

    DROP TEMPORARY TABLE IF EXISTS temp_table;
    CREATE TEMPORARY TABLE temp_table
    SELECT 1 AS v1, 2 AS v2;
    SELECT * FROM temp_table;

If I execute use.sql, a tab with create.sql is opened and I get this errror message:

    SQL-Fehler [1146] [42S02]: Table 'bn3.temp_table' doesn't exist
But apparently create.sql isn't executed - the newly created tab doesn't show a result. And in this tab create.sql can't be edited - I can select the text but not edit it.

**Comment by kseniiaguzeeva** [comment]
Please disable function Open separate connection for each Editor. It should help.
![dbeaver_YjuENu2nih](https://user-images.githubusercontent.com/51405061/85268977-ec81d780-b47f-11ea-8e6c-ba79e40b9fe9.png)

**Comment by boesi** [comment]
@kseniiaguzeeva Thanks at lot, that was the hint I needed. I had not noticed this option before, although it seams to exist way longer  than I use dbeaver.
